aviator.yml

```diff
@@ -0,0 +1,16 @@
+spruce:
+  - base: ci/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/
+        regexp: ".*.yml"
+      - with_in: ci/jobs/
+        regexp: ".*.yml"
+    to: aviator_pipeline.yml
+fly:
+  name: repo-template-docker
```

Comment:
is this going to be changed by the initial commit script in the github-configs repo? If so, should it follow the `example` naming for the find/replace?